### PR TITLE
Security changes

### PR DESCRIPTION
**Pull request by LAShemilt**

### CODE REVIEW

**Comment by LAShemilt** [comment]
Here I have taken out the LDAP changes and am using the flask_login UserMixin to do the authentication on the users. I also use werkzeug to hash the passwords in memory for some extra security.
Let me know if you like the idea. There is a little more work to do before this can be merged.

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -42,23 +42,8 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with nose 
       env:
-         TEST_USERNAME: fry
-         TEST_PASSWORD: fry
-         TEST_USERNAME2: zoidberg
-         TEST_PASSWORD2: zoidberg
+         USER_SETUP_JSON: tests/users.json
          S3_ENDPOINT_URL: 'http://s3.amazonaws.com'
```
Comment (tschoonj) [inline .github/workflows/python-app.yml]:
```suggestion
         S3_ENDPOINT_URL: 'https://s3.amazonaws.com'
```

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -42,23 +42,8 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with nose 
       env:
-         TEST_USERNAME: fry
-         TEST_PASSWORD: fry
-         TEST_USERNAME2: zoidberg
-         TEST_PASSWORD2: zoidberg
+         USER_SETUP_JSON: tests/users.json
```
Comment (tschoonj) [inline .github/workflows/python-app.yml]:
```suggestion
         USER_SETUP_JSON: hancock/tests/users.json
```

Not sure but could perhaps fix your CI failure?

File: hancock/tests/test_hancock.py [hunk 3]
```diff
@@ -78,6 +78,7 @@ def setUp(self) -> None:
         creds = self.session.get_credentials()
         app.config['ACCESS_KEY'] = creds.access_key
         app.config['SECRET_ACCESS_KEY'] = creds.secret_key
+        app.config['USER_SETUP_JSON'] = 'test_users.json'
```
Comment (tschoonj) [inline hancock/tests/test_hancock.py]: Does this file exist? Why override the one you get from the environment?

File: requirements.txt [hunk 4]
```diff
@@ -2,11 +2,12 @@ boto3
 botocore
 flask
 flask-jwt-extended
-flask-ldap3-login
 flask-restx
 python-dotenv
 redis
 uwsgi
 requests
 mock==4.0.2
```
Comment (tschoonj) [inline requirements.txt:10]: version constraints make baby Jesus cry 😉

Comment (LAShemilt) [reply requirements.txt:10]: oh yes - that is a moto problem. I spent many hours on it ...

**Comment by LAShemilt** [comment]
closes #7

**Pull request merged**